File: quiz/week11_2/devowel_file.pl
#!/usr/bin/perl -w

die "Usage: $0: <file>\n" if @ARGV != 1;
$filename = $ARGV[0];

open OLD,'<',$filename or die;
while($line = <OLD>){
	$line =~ s/[aeiou]//ig;
	push @lines, $line;
}
close OLD;

open NEW,'>',$filename or die;
print NEW, @lines;
close NEW;

File: quiz/week11_2/devowel_file.py
#!/usr/bin/python3
import re,sys

filename = sys.argv[1]

with open(filename) as f:
	file_contents = f.read()

new_file_contents = re.sub(r'[aeiou]','',file_contents, flags=re.I)
with open(filename,'w') as f:
	f.write(new_file_contents)

